### PR TITLE
IDEX-3243 Use the che-host property defined when starting the docker container

### PR DESCRIPTION
**Pull request by benoitf**
Change-Id: I39627f243dd84c7650a8d4feb3870b72d9e7406a
Signed-off-by: Florent BENOIT fbenoit@codenvy.com

### CODE REVIEW

**Comment by benoitf** [comment]
needs https://github.com/codenvy/che-plugins/pull/337

File: assembly-sdk/pom.xml [hunk 1]
```diff
@@ -29,6 +29,12 @@
             <type>zip</type>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.che.dashboard</groupId>
```
Comment (garagatyi) [inline assembly-sdk/pom.xml]: Is it needed for detecting api endpoint?

File: assembly-sdk/src/assembly/assembly.xml [hunk 2]
```diff
@@ -31,6 +31,15 @@
         <dependencySet>
             <useProjectArtifact>false</useProjectArtifact>
             <unpack>false</unpack>
+            <outputDirectory>webapps</outputDirectory>
```
Comment (garagatyi) [inline assembly-sdk/src/assembly/assembly.xml]: Is it needed in this PR?

**Comment by benoitf** [comment]
merged two PR in one. will move out

**Comment by benoitf** [comment]
the other part was now in another PR https://github.com/codenvy/che/pull/82

**Comment by skabashnyuk** [comment]
LGTM

**Pull request merged**